### ISSUE TITLE
finbar-cli library not loaded

### ISSUE THREAD

**Issue by Ptujec** [roeybiran/finbar-issues#19]
I get the following response when trying to use finbar-cli:

```
Library not loaded: @rpath/Sparkle.framework/Versions/B/Sparkle
  Referenced from: <BE7F9E95-4F32-3E74-80C6-72F0707A16C5> /Applications/Finbar.app/Contents/MacOS/finbar-cli
  Reason: no LC_RPATH's found
```

**Comment by roeybiran** [comment]
Thanks for reporting! I’ll check this ASAP.

**Comment by roeybiran** [comment]
Fixed in v1.13.5 released today.
